### PR TITLE
New way to iterate resultsets test m tree

### PR DESCRIPTION
**Pull request by rcapecce**
Related #3723

### CODE REVIEW

File: base/src/org/compiere/util/DB.java [hunk 1]
```diff
@@ -2597,4 +2597,51 @@ public static Timestamp getCurrentTimeFromDatabase() {
...
+
+	/**
+	 * Execute ResultSet without worrying about handling closure of database objects
+	 * @param trxName Database transaction
+	 * @param query Query to run in the PreparedStatement
+	 * @param parameters Parameters of the query
+	 * @param resultSetRunnable Execution of the ResultSet
+	 * @throws SQLException Handles the correct closing of persistence objects, not the thrown exception
+	 */
+    public static void runResultSet(String trxName, String query, Object[] parameters, ResultSetRunnable resultSetRunnable) throws SQLException {
```
Comment (yamelsenih) [inline base/src/org/compiere/util/DB.java:2620]: Can you declare it `Object[] parameters` as this `Object... parameters` just add at end of parameters and it is a array very dynamic

**Review by yamelsenih — CHANGES_REQUESTED** [review]

**Review by rcapecce — COMMENTED** [review]
Hi @yamelsenih I remove this implementation and use the parammeters to the endo of the signature

**Review by rcapecce — COMMENTED** [review]
Can you review this new implementation?

**Comment by e-Evolution** [comment]
This pull request was implement in join with #3735

**Pull request closed without merging**